plugins/processors/t128_transform/README.md

```diff
@@ -6,23 +6,42 @@ The `t128_transform` transforms metrics based on the difference between two obse
 
 ```toml
 [[processors.t128_transform]]
-  ## If more than this amount of time passes between data points, the
-  ## previous value will be considered old and the value will be recalculated
-  ## as if it hadn't been seen before. A zero expiration means never expire.
-  # expiration = "0s"
+  ## For 'rate' and 'diff', if more than this amount of time passes between
```

Comment:
I think we have a mix of spaces and tabs in this file. Maybe convert one to the other. In VSCode, that would be `Cmd+Shift+p` and then `Convert Indentation to Spaces/Tabs`.